src/server/components/dashboard/controllers.ts

```diff
@@ -56,6 +57,7 @@ export async function startRouteController(
       req,
     });
   } catch (error) {
+    logger.error(`startRouteController Error: ${(error as Error).message}`);
     next(error);
```

Comment:
the main benefit of adding an error here is you get a more descriptive context without having to trawl through code.  You could chain the error something by adding a rootCause property to our custom HttpError and create a constructor to pass a message and error.stack so you'd end up with something like: 


```
const new HttpError("Error processing start route", error.stack);
next(error);
```


the error handler middleware can then use error.stack when it logs the error out so it can be used for tracing / support investigation.

Reply:
Hey @alisalaman-cyb I'm not exactly sure what you mean. From what you've said sounds like you want to expose the error.stack by passing it down as an argument? If that's the case I don't think that's necessary since the extended Error class already captures the stack

```ts
// line 4 from src/server/middlewares/err-handlers.ts
export class HttpException extends Error {
  ...
```

It's possible to expose the stack by just calling `error.stack`

<img width="1379" alt="Screenshot 2022-10-27 at 14 01 03" src="https://user-images.githubusercontent.com/1377253/198293407-b336a6df-ff49-412b-8c29-a24feef5097b.png">

Please let me know if I'm misunderstanding your comment.